src/components/dashboard/DashboardOtherAssets.js

```diff
@@ -7,206 +7,206 @@ import { Grid, List } from 'semantic-ui-react'
-                  <List.Icon />
-                  <List.Content>
-                     <List.Header as='h2' className='color-charcoal-grey'>
-                        2.5897
-                     </List.Header>
-                     <List.Description as='h5' className='color-brown-grey'>
-                        XMR
-                     </List.Description>
-                  </List.Content>
-               </List.Item>
-            </List>
-         </Grid.Column>
-         <Grid.Column>
-            <List horizontal>
-               <List.Item>
-                  <List.Icon />
-                  <List.Content>
-                     <List.Header as='h2' className='color-charcoal-grey'>
-                        1.2589
-                     </List.Header>
-                     <List.Description as='h5' className='color-brown-grey'>
-                        ZIL
-                     </List.Description>
-                  </List.Content>
-               </List.Item>
-            </List>
-         </Grid.Column>
-      </Grid.Row>
-   </CustomGrid>
+    <CustomGrid>
+        <Grid.Row className='other-assets-title' as='h6'>
+            OTHER ASSETS
+        </Grid.Row>
+        <Grid.Row columns={6} className='other-assets'>
+            <Grid.Column>
+                <List horizontal>
+                    <List.Item>
+                        <List.Icon />
+                        <List.Content>
+                            <List.Header as='h2' className='color-charcoal-grey'>
+                                12.125
+                            </List.Header>
+                            <List.Description as='h5' className='color-brown-grey'>
+                                ETH
+                            </List.Description>
+                        </List.Content>
+                    </List.Item>
+                </List>
+            </Grid.Column>
+            <Grid.Column>
+                <List horizontal>
+                    <List.Item>
+                        <List.Icon />
+                        <List.Content>
+                            <List.Header as='h2' className='color-charcoal-grey'>
+                                3.2501
+                            </List.Header>
+                            <List.Description as='h5' className='color-brown-grey'>
+                                XRP
+                            </List.Description>
+                        </List.Content>
+                    </List.Item>
+                </List>
+            </Grid.Column>
+            <Grid.Column>
+                <List horizontal>
+                    <List.Item>
+                        <List.Icon />
+                        <List.Content>
+                            <List.Header as='h2' className='color-charcoal-grey'>
+                                8.0987
+                            </List.Header>
+                            <List.Description as='h5' className='color-brown-grey'>
+                                ZEC
```

Comment:
Looks like there's a lot of IDE spacing in this PR?

Reply:
Whitespace changes are by design if I understood you correctly.

Reply:
ah yes, I see the commit names now.